include/swift/AST/ModuleDependencies.h

```diff
@@ -1129,6 +1129,9 @@ class ModuleDependenciesCache {
   std::string scannerContextHash;
   /// The location of where the built modules will be output to
   std::string moduleOutputPath;
+  /// The timestamp of the beginning of the scanning query action
+  /// using this cache
+  llvm::sys::TimePoint<> scanInitializationTime;
```

Comment:
Nit: make this `const` if not intended to update.

Reply:
Done. Thanks.